packages/synapse-react-client/src/lib/containers/entity/annotations/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -273,13 +269,7 @@ export const SchemaDrivenAnnotationEditor = (
             }}
           >
             {validationError && (
-              <Alert
-                className="ErrorList"
-                dismissible={false}
-                show={true}
-                variant="danger"
-                transition={false}
-              >
+              <Alert severity="error" sx={{ mt: 2 }}>
```

Comment:
main | feature
:---:|:---:
<img width="950" alt="main_SchemaDrivenAnnotationEditor_ErrorAlert" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/7e20fef2-9f5d-40a6-bccc-baf9eb6b6c03"> | <img width="983" alt="feature_SchemaDrivenAnnotationEditor_ErrorAlert" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/99ecaae7-48ff-496b-b1cb-2b8394767672">

Reply:
Also apply styling from _annotations.scss directly to the Alert